react-client/src/components/App.jsx

```diff
@@ -180,6 +180,8 @@ class App extends React.Component {
   }
 
   postLike (wine, likeOrDislike) {
+    var context = this;
```

Comment:
Just wondering if we really need 'context' here, since the function is bound to 'this' inside the constructor.